cloud/azure/variables.tf

```diff
@@ -0,0 +1,52 @@
+# Global
+
+variable "suffix" {
+  description = "Optional suffix to identify and avoid duplication of unique resources"
+  type        = string
+  default     = ""
+}
+
+# Azure Integration specific
+
+variable "enabled" {
+  description = "Whether the Azure integration is enabled"
+  type        = bool
+  default     = true
+}
+
+variable "poll_rate" {
+  description = "AWS poll rate in seconds (60 or 300)"
```

Comment:
@xp-1000 why this messy change?

Reply:
just homogenize with other modules code and behavior